packages/fx-core/resource/package.nls.json

```diff
@@ -207,6 +207,8 @@
   "error.generator.DownloadSampleApiLimitError": "Unable to download sample due to throttling. Retry later after rate limit reset (This may take up to 1 hour). Alternatively, you can go to %s to git clone the repo manually",
   "error.generator.DownloadSampleNetworkError": "Unable to download sample due to network error. Check your network connection and retry. Alternatively, you can go to %s to git clone the repo manually",
   "error.generator.ParseUrlError": "Unable to parse url %s",
+  "error.copilotPlugin.openAiPluginManifest.CannotGetLogoError": "Unable to get logo from '%s'. Please update '%s' manually if you want to use same logo for your Copilot plugin.",
+  "error.copilotPlugin.openAiPluginManifest.CannotGetManifest": "Unable to get OpenAI plugin manifest from '%s'.",
```

Comment:
Better for us not to convert the logo/icon as the Teams app has specific icon sizing requirements. We can follow practice for other templates where we use a default one and developers can change by themselves.